lilypad/models/fn_params.py

```diff
@@ -1,24 +1,63 @@
 """Provider call params models"""
 
-from typing import Any
+from typing import Literal
 
 from pydantic import BaseModel
 
 from lilypad.server.models import FnParamsBase, Provider
 
 
+class ResponseFormat(BaseModel):
+    """Response format model."""
+
+    type: Literal["text", "json_object", "json_schema"]
+
+
+class OpenAICallArgsCreate(BaseModel):
+    """OpenAI call args model."""
+
+    max_tokens: int
```

Comment:
not all of these are required fields?

Reply:
they arent required but they have default values, so they are always set to default in the UI

Reply:
but many of them can be null/none? the api doesn't require them